frontend/screens/ProfileScreen/index.js

```diff
@@ -6,16 +6,81 @@ import {
 import styles from './styles';
 import { getUser } from '../../controllers/UserController';
 import * as Screen from '../../navigation/tab_navigator/stacks/profile/screen-names';
+import ScheduleComponent from '../../components/ScheduleComponent/index'
+import Day from '../../enums/Day';
+import moment from 'moment';
+import Schedule from '../../models/Schedule';
 
 function Profile({ navigation }) {
   const [user, setUser] = useState(JSON.parse('{"username" : "", "email" : "", "displayName" : ""}'));
 
+  const schedule = new Schedule();
+  schedule.availability = [
```

Comment:
This is just dummy data, right?

Reply:
yeaah

Reply:
Yes this is dummy data, and I think we are removing the Schedule model as well. I'm going to add in controllers and such for adding things to a user's schedule then you can use it to pull the schedule from the database. I changed my schedule component for these changes in a different branch. You can push the dummy data for now then change them later once I'm done with the controllers.